extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -200,6 +248,7 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
       },
       () => {
         this.styleTagText_();
+        this.shouldTagFlip_(this.storeService_.get(StateProperty.PAGE_SIZE));
```

Comment:
This method is being called in response to both the `SHOPPING_DATA` and `PAGE_SIZE` actions. Is this necessary?

Reply:
Good point - needed to add `true /** callToInitialize */` to get it to work.